components/skeleton/Skeleton.tsx

```diff
@@ -7,10 +7,10 @@ import Element from './Element';
 import SkeletonAvatar, { AvatarProps } from './Avatar';
 import SkeletonButton from './Button';
 import SkeletonInput from './Input';
+import SkeletonImage from './Image';
 
 /* This only for skeleton internal. */
-interface SkeletonAvatarProps extends Omit<AvatarProps, 'active'> {
-}
+interface SkeletonAvatarProps extends Omit<AvatarProps, 'active'> {}
```

Comment:
Why this changed? I have on idea why make this change. Maybe your lint have some problem.

Reply:
It seems look like the code style formatter of ant design's precommit hook?